lib/travis/build/addons/firefox.rb

```diff
@@ -68,13 +68,27 @@ def export_source_url
               'firefox-beta-latest'
             when 'latest-esr'
               'firefox-esr-latest'
+            when 'latest-dev'
+              'firefox-aurora-latest'
```

Comment:
Whoops, I thought I replied to this. Aurora is the term for the branch that holds firefox developer edition. Developer edition is usually any features that have been on nightly for ~6 weeks.